pkg/services/dashboards/database/database.go

```diff
@@ -505,6 +507,25 @@ func saveDashboard(sess *sqlstore.DBSession, cmd *models.SaveDashboardCommand) e
 		}
 	}
 
+	var entityEvent *store.EntityEvent
+	if dash.IsFolder {
+		entityEvent = &store.EntityEvent{
+			EventType: store.EntityEventTypeUpdate,
+			EntityId:  store.CreateDatabaseEntityId(dash.Uid, dash.OrgId, store.EntityTypeFolder),
+			Created:   time.Now().Unix(),
+		}
+	} else {
+		entityEvent = &store.EntityEvent{
+			EventType: store.EntityEventTypeUpdate,
+			EntityId:  store.CreateDatabaseEntityId(dash.Uid, dash.OrgId, store.EntityTypeDashboard),
+			Created:   time.Now().Unix(),
+		}
+	}
+	_, err = sess.Insert(entityEvent)
```

Comment:
nit: could be extracted to thats used in both places

```suggestion
	_, err = sess.Insert(createEntityEvent(dash, store.EntityEventTypeUpdate))
```

Reply:
Done in 78c7b881e1ae857d65376782f24941a30c9e6930